lib/query-params/query-params.js

```diff
@@ -28,7 +28,9 @@ module.exports = (typeFormat, queryParams) => {
 
   // Objects
   Object.assign(result.filter.objects, queryParamsObjects(typeFormat, queryParams));
-
+  if (result.filter.objects.categories) {
```

Comment:
@SimonLab please consider adding more checks to avoid a `ReferenceError`:
```js
if (result && result.filter result.filter.objects && result.filter.objects.categories)
```
This will not create a big "tax" on perf but it will avoid a unrecoverable error.